screens/LoginScreen.js

```diff
@@ -393,9 +330,9 @@ class LoginScreen extends React.Component {
         position="center"
       />
     );
-    const { domainValidation, userValidation, passwordValidation } = this.state;
+    /* const { domainValidation, userValidation, passwordValidation } = this.state;
```

Comment:
Why is this commented out? This field validation should be uncommented and should still be working as implemented in the development branch.

Reply:
Sorry for that, i will uncomment the logic an test it.